sdk/core/azure-core/tests/test_rest_request_backcompat.py

```diff
@@ -304,7 +304,7 @@ def test_request_formdata_body_files(old_request, new_request):
     new_request.set_formdata_body({"fileName": "hello.jpg"})
 
     assert old_request.data == new_request.data == None
-    assert old_request.files == new_request.files == new_request.content == {"fileName": (None, "hello.jpg")}
+    assert old_request.files == new_request.files == new_request.content == {"fileName": (None, "hello.jpg", "application/octet-stream")}
```

Comment:
we purposefully always return with default content type `application/octet-stream` so we can have standard behavior cc @lmazuel